src/test/resources/features/prisoner_search.feature

```diff
@@ -96,17 +96,14 @@ Feature: Prisoner Search
       | A1476AE    | 0             |           |
       | A1181MV    | 1             | O'VAUGHAN |
 
-  Scenario Outline: Search for prisoners with specified offender number using simple unprotected endpoint
-    Given a user has a token name of "GLOBAL_SEARCH"
+  Scenario Outline: Search for prisoners with specified offender numbers using simple endpoint
+     Given a user has a token name of "GLOBAL_SEARCH"
     When a search is made for prisoners with offender numbers of "<offenderNos>" using simple endpoint
     Then "<numberResults>" prisoner records are returned
     And the prisoners last names match "<lastNames>"
 
     Examples:
       | offenderNos        | numberResults | lastNames       |
-      | A1234AC            | 1             | BATES           |
-      | A1476AE            | 0             |                 |
-      | A1181MV            | 1             | O'VAUGHAN       |
```

Comment:
Removed tests associated to endpoint with new role

Reply:
if there are no tests now for `/api/prisoners/XXX` can that step definition now be removed?